notebooks/02_model/lightgcn_deep_dive.ipynb

```diff
@@ -0,0 +1,807 @@
+{
```

Comment:
<p>maybe here say:</p><p> </p><p>This notebook serves as an introduction to LightGCN, which is an simple, linear and neat Graph Convolution Network (GCN) model for recommendation.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>